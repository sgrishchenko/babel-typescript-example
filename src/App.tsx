import React from 'react';
import { hot } from 'react-hot-loader';

export const App = () => {
  const text: string = 'Hello!!!';
  return (
    <div>{text}</div>
  );
};

export default hot(module)(App);
