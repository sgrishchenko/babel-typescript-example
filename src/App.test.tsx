import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App Component', () => {
  test('should contains main container', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
