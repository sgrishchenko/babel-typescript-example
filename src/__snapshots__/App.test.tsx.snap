// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['App Component should contains main container 1'] = `
<div>
  Hello!!!
</div>
`;
